packages/app/app/actions/player.js

```diff
@@ -36,6 +36,14 @@ export function togglePlayback(currentState) {
   };
 }
 
+export function resetPlayer() {
+  return dispatch => {
+    dispatch(updatePlaybackProgress(0,0));
+    dispatch(pausePlayback());
```

Comment:
commenting this out allows the test the pass successfully because ipcrenderer is undefined